src/resources/views/crud/fields/video.blade.php

```diff
@@ -153,13 +153,14 @@
 
         var fetchYouTube = function( videoId, callback, apiKey ){
 
-            var api = 'https://www.googleapis.com/youtube/v3/videos?id='+videoId+'&key='+apiKey+'&part=snippet';
+            var api = `https://www.googleapis.com/youtube/v3/videos?id=${videoId}&key=${apiKey}&part=snippet,contentDetails`;
 
             var video = {
                 provider: 'youtube',
-                id: null,
+                id: videoId,
```

Comment:
Quick note on this;
By having the `videoId` by default here this becomes a breaking change, because until now, the `video.id` was only being filled when the request was successful, and was filled along with the other attributes.

By having it by default, when the request fails, the video field may be filled with the `id` only.

Reply:
Good catch @promatik . How can we make this non-breaking then? It'd be better to not have any breaking changes inside fields, even in 4.2 - otherwise the upgrade guide will get pretty long.

Reply:
@tabacitu I've pushed a commit to avoid the breaking change 👌